group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -490,6 +490,11 @@ GroupMetadataManager build() {
      */
     private MetadataImage metadataImage;
 
+    /**
+     * The topic hash value by topic name.
+     */
```

Comment:
It would be great if we could expand the comment to explain how we maintain this cache.